rtwo/drivers/openstack_user.py

```diff
@@ -37,25 +40,34 @@ def lc_driver_init(self, lc_driver, *args, **kwargs):
         return manager
 
     def __init__(self, *args, **kwargs):
+        self.built_using_token=False
         self.keystone, self.nova, self.swift = self.new_connection(*args, **kwargs)
-        auth_version = kwargs.get('version','v2.0')
-        if '2.0' in auth_version:
-            self.version = 2
-        elif '3' in auth_version:
-            self.version = 3
-        else:
-            raise Exception("Could not determine a version of openstack_user based on %s" % auth_version)
-
+        if not kwargs.has_key('ex_force_auth_token'): 
```

Comment:
The driver should not care if it was built using tokens, or if it was initialized with ex_force_auth_token, we are deferring those questions to the individual connections, `swift, nova, keystone` to determine how they will be created with the kwargs passed in.